packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -1,22 +1,40 @@
 import { Box, Flex, Grid } from '@strapi/design-system';
+import { useIntl } from 'react-intl';
 
+import { useDoc } from '../../../hooks/useDocument';
 import { EditLayout } from '../../../hooks/useDocumentLayout';
 
 import { InputRenderer } from './InputRenderer';
 
 interface FormLayoutProps extends Pick<EditLayout, 'layout'> {}
 
 const FormLayout = ({ layout }: FormLayoutProps) => {
+  const { formatMessage } = useIntl();
+  const { model } = useDoc();
+
+  const getTranslatedLabel = (label: string) => {
+    return formatMessage({
+      id: `content-manager.${model}.${label}`,
```

Comment:
This is much better but I think we could still have collisions 🤔 how about `content-manager.[something].${model}.${label}`, with something like field in the middle?

Reply:
`content-manager.content-types.${model}.${label}`? wdyt? All ears for another suggestion, before it was containers but I don't like that

Reply:
sounds good!